corehq/sql_db/operations.py

```diff
@@ -100,24 +105,28 @@ class RawSQLMigration(object):
 
         migrator = RawSQLMigration(('base', 'path'), {'variable': 'value'})
         migrator.get_migration('sql_template.sql')
+
+    The reverse migration will be a no-op by default. To make a
+    non-reversible migration, set the reverse template to None:
+
+        migrator.get_migration('sql_template.sql', None)  # non-reversible
     """
 
     def __init__(self, base_path_tuple, template_context=None):
         self.template_context = template_context
         self.base_path = os.path.join(*base_path_tuple)
 
-    def get_migration(self, forward_template, reverse_template=Ellipsis, testing_only=False):
+    def get_migration(self, forward_template, reverse_template=NOOP, testing_only=False):
         if testing_only and not settings.UNIT_TESTING:
             return noop_migration()
 
         forward_path = os.path.join(self.base_path, forward_template)
 
-        if reverse_template is Ellipsis:
-            # reverse could be None to make the migration non-reversible
-            reverse_template = 'SELECT 1'  # noop reverse
```

Comment:
For default no-op migrations the reverse migration path was set to `os.path.join(self.base_path, reverse_template)` -> `"some/base/path/SELECT 1"`, which is clearly wrong, but that was passed over because of a bug in `RunSqlLazy._render_template`.